contracts/EtherTimelock.sol

```diff
@@ -0,0 +1,37 @@
+pragma solidity ^0.4.17;
+
+/**
+ * @title EtherTimelock
+ * @dev EtherTimelock is a contract that will allow a
+ * beneficiary to extract ether sent to it after a given release time
+ */
+contract EtherTimelock {
+    //beneficiary of funds
+    address public beneficiary;
+
+    //timestamp when funds are able to be released
+    uint64 public releaseTime;
+
+    function EtherTimelock(address _beneficiary, uint64 _releaseTime) {
+        require(_releaseTime > now);
+
+        beneficiary = _beneficiary;
+        releaseTime = _releaseTime;
+    }
+
+   /**
+   * @notice Transfers funds held by timelock to beneficiary.
+   */
+    function release() public { 
+        require(now > releaseTime);
+
+        uint256 amount = this.balance;
```

Comment:
why assigning `this.balance` to amount? `this.balance` can be used in the require and transfer, I think it would be simpler.

Reply:
I had seen some other contracts do this, I agree on the simpler part of just using this.balance.